<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectNewDialog</class>
 <widget class="QDialog" name="ProjectNewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Project Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="projectName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="projectPath"/>
       </item>
       <item>
        <widget class="QToolButton" name="toolFindProjectPath">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Project file</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="projectFileText">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Template</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="templateFile">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolLoadTemplate">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectNewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectNewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ProjectNewDialog</receiver>
   <slot>refreshProjectName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectPath</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ProjectNewDialog</receiver>
   <slot>refreshProjectName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>templateFile</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>ProjectNewDialog</receiver>
   <slot>refreshProjectName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshProjectName()</slot>
 </slots>
</ui>
